addons/website_blog/views/website_blog_templates.xml

```diff
@@ -1,12 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-<template id="_assets_primary_variables" inherit_id="web._assets_primary_variables">
-    <xpath expr="//link[last()]" position="after">
-        <link rel="stylesheet" type="text/scss" href="/website_blog/static/src/scss/primary_variables.scss"/>
-    </xpath>
-</template>
```

Comment:
@sdegueldre You remove this in the third commit but remove the file in the second :/ Fixed.